tests/test_logger.py

```diff
@@ -3,7 +3,7 @@
 from stable_baselines.logger import make_output_format, read_tb, read_csv, read_json, _demo
 
 
-KEY_VALUES = {'test': 1, 'b': -3.14, '8': 9.9}
+KEY_VALUES = {'test': 1, 'b': -3.14, '8': 9.9, 'l': [1, 2]}
```

Comment:
is the conversion to np array done automatically?

Reply:
sorry, was originally targeting python lists, I will add a np.array as well.